src/common/dom-helpers/dom-helpers.module.ts

```diff
@@ -0,0 +1,15 @@
+import {NgModule} from '@angular/core';
```

Comment:
blur directive should be in focus folder, we try to have feature-based folder structure

Reply:
according to this remark, the blur directive has been moved into the 'focus' folder in the commit: [blur directive, dom base class: refactoring](https://github.com/qgrid/ng2/pull/39/commits/bb9b43a5374e93ada17ace5c6f028fd5aa7ac651).